find_ideal_patchsize_and_magnification.py

```diff
@@ -0,0 +1,260 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:light
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.14.1
+#   kernelspec:
+#     display_name: Python [conda env:Pytorch]
+#     language: python
+#     name: conda-env-Pytorch-py
+# ---
+
```

Comment:
For sure you want to add your name + email address here with the date, so that you can receive credit for your work